Tensile/ClientWriter.py

```diff
@@ -551,21 +594,12 @@ def dataInitParams(problemType):
             ('init-alpha', dataInitName(initAlpha)),
             ('init-beta',  dataInitName(initBeta))]
 
-def writeClientConfig(forBenchmark, solutions, problemSizes, stepName, stepBaseDir, newLibrary, codeObjectFiles, tileAwareSelection = False):
-
-    if tileAwareSelection:
-      filename = os.path.join(globalParameters["WorkingPath"], "ClientParameters_Granularity.ini")
-    else:
-      filename = os.path.join(globalParameters["WorkingPath"], "ClientParameters.ini")
-
+def writeClientConfigImpl(problemSizes, problemType, sourceDir, codeObjectFiles, resultsFileName, parametersFilePath):
```

Comment:
I like this separation.  I wonder if we could come up with a better name than just adding 'Impl' to the function name?

Reply:
How about writeClientConfigIni?